rundeckapp/grails-spa/packages/ui-trellis/src/app/components/home/HomeCardList.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <div class="card">
+    <div id="project-list" class="card-content">
+      <div>
+        <HomeSearchBar
+            :placeholder="$t('page.home.search.projects.input.placeholder')"
+            v-model="search"
+            @on-focus="hideResults"
+            @on-enter="handleSearch()"
+            @on-blur="handleSearch()"
+        />
+        <ui-socket location="table-header" section="home" />
+        <p data-test="searchResults" class="alert alert-info" v-if="showSearchResults && loadedProjectNames">
+          <span :class="{ 'text-white': searchResultsCount>0, 'text-warning': searchResultsCount<1 }">
+            {{ $tc('page.home.search.project.title', searchResultsCount) }}
+          </span>
+        </p>
+        <offset-pagination
+            :pagination="pagination"
+            @change="changePageOffset"
+            :showPrefix="true"
+        ></offset-pagination>
+      </div>
+
+      <div class="project_list_header">
+        <div class="row row-border-top p-4">
+          <div class="col-sm-6 col-md-8">
+            <p class="text-lg">
+              {{ $t('home.table.projects') }}
+            </p>
+          </div>
+          <div class="col-sm-6 col-md-2">
+            <p class="text-lg">
+              {{ $t('home.table.activity') }}
+            </p>
+          </div>
+          <div class="col-sm-12 col-md-2 col-last">
+            <p class="text-lg">
+              {{ $t('home.table.actions') }}
+            </p>
+          </div>
+        </div>
+      </div>
+      <HomeBrowser v-if="searchResultsCount > 0" :key="searchResultsCount" :projects="resultsPage" :allProjects="projects" :loaded="loadedProjectNames" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import {defineComponent, PropType} from 'vue'
+import HomeSearchBar from "./HomeSearchBar.vue";
+import HomeBrowser from "./HomeBrowser.vue";
+import {Project} from "@/app/components/home/types/projectTypes";
+import UiSocket from "@/library/components/utils/UiSocket.vue";
+import {getRundeckContext} from "@/library";
+import OffsetPagination from "@/library/components/utils/OffsetPagination.vue";
+
+export default defineComponent({
+  name: "HomeCardList",
+  components: {
+    UiSocket,
+    HomeSearchBar,
+    HomeBrowser,
+    OffsetPagination
+  },
+  props: {
+    loadedProjectNames: {
+      type: Boolean,
+      default: false
+    },
+    projects: {
+      type: Array as PropType<Project[]>,
+      default: () => []
+    }
+  },
+  data() {
+    return {
+      search: "",
+      filteredProjects: [] as Project[],
+      favoriteProjectNames: [] as Project[],
+      resultsPage: [] as Project[],
+      showSearchResults: false,
+      filterFavoritesOnly: false,
+      eventBus: getRundeckContext().eventBus,
+      pagination: {
+        offset: 0,
+        max: 30,
+        total: -1
+      }
```

Comment:
the home.next.gsp (and home.gsp) defines some paging parameters which could be used here, maybe just the `pagingMax` for now